charts/logzio-telemetry/values.yaml

```diff
@@ -944,4 +997,61 @@ opencost:
+        custom: container_oom_events_total
+      drop:
+        custom: 
+    # for applications pipeline: applications job
+    applications:
+      keep:
+        custom: avalanche_metric_mmmmm_119_0|avalanche_metric_mmmmm_120_0
+      drop:
+        custom: 
```

Comment:
I think wee need to add a comment about the expect input here, It can be thought as an yaml object its not clear that you need to input regex

Reply:
Ive added a comment in the next to "prometheusFilters" (the top key), ill add it also to each of these values.